lua/entities/gmod_wire_expression2/core/custom/applyforce.lua

```diff
@@ -0,0 +1,196 @@
+E2Lib.RegisterExtension("applyforce", false, "Allows E2 chips to applyforce", "Allows E2 chips to applyforce")
+
+local sbox_E2_ApplyForce = CreateConVar( "sbox_E2_ApplyForce", "1", FCVAR_ARCHIVE ) -- 1: Everyone, 2: Only Admins, 3: Only Superadmins
+
+function ApplyForceValidAction()
```

Comment:
And if he does make if "local", as you say, then he will need to DUPLICATE it, because he also uses it in gates... So, imo, it doesn't belong here.